test/fixtures/get-orderbook-response.json

```diff
@@ -169,6 +169,26 @@
         "direction": "buy",
         "quantity": {
           "currency": "USD",
+          "value": "0.003967400879423823",
```

Comment:
Ah, I see, https://github.com/ripple/ripple-lib/blob/develop/test/fixtures/book-offers-response.js#L228 (one little typo there, by the looks )

And: https://github.com/ripple/ripple-lib/blob/develop/test/fixtures/book-offers-response.js#L7-L13
The results were taken from real fixtures, but they are flipped, such that the real life ratios are inverted.